src/analyses/accessAnalysis.ml

```diff
@@ -34,14 +57,28 @@ struct
     vulnerable := 0;
     unsafe := 0
 
+  let side_vars ctx lv_opt ty =
+    match lv_opt with
+    | Some (v, _) ->
+      let d =
+        if !GU.should_warn then
+          G.create_vars (V0Set.singleton (lv_opt, ty))
+        else
+          G.bot () (* HACK: just to pass validation with MCP DomVariantLattice *)
```

Comment:
Could this hack hide unreached fixed-point problems?

Reply:
Not really. The existing global invariant based access collecting already does the same below. This bottom side effect has absolutely no bad effect. The only problematic place would be the overridden `leq` implementation during `postsolving`. But that doesn't affect the fixpoint errors for any other analyses.